telegram/error.py

```diff
@@ -118,6 +118,13 @@ def __init__(self, message: Optional[str] = None) -> None:
 class NetworkError(TelegramError):
     """Base class for exceptions due to networking errors.
 
+    Tip:
+        This exception (and its subclasses) usually originate from the networking backend
+        used by :class:`~telegram.request.HTTPXRequest`/a custom implementation of
```

Comment:
```suggestion
        used by :class:`~telegram.request.HTTPXRequest`, a custom implementation of
```

Reply:
no, I think he meant "or".
```suggestion
        used by :class:`~telegram.request.HTTPXRequest`, or a custom implementation of
```